runtime/tests/Makefile

```diff
@@ -1,4 +1,4 @@
-TESTPROGS=string_hash string_cmp string_concat string_len immediate_string tagged_ptr
+TESTPROGS=string_hash string_cmp string_concat string_len immediate_string tagged_ptr float_print
```

Comment:
Also needs adding to .gitignore

Reply:
Fixed. May I create an issue to build all artifacts in a dir like `target` or `build`?

Reply:
By all means create an issue, but I think it's better to solve this later:

- we need `bal` command to allow control over `target` dir
- we will want to switch to `cmake` eventually, which will support this for free